spec-main/api-app-spec.ts

```diff
@@ -941,10 +941,13 @@ describe('app module', () => {
         expect(entry.cpu).to.have.ownProperty('idleWakeupsPerSecond').that.is.a('number')
 
         if (process.platform !== 'linux') {
+          expect(entry.memory).to.have.property('workingSetSize').that.is.a('number')
```

Comment:
could we also test lower bounds for this? i.e we could safely assume this might be a number that's greater than 0?